paddlenlp/metrics/glue.py

```diff
@@ -26,7 +26,37 @@
 
 class AccuracyAndF1(Metric):
     """
-    Encapsulates Accuracy, Precision, Recall and F1 metric logic.
+    This class encapsulates Accuracy, Precision, Recall and F1 metric logic,
+    and `accumulate` function returns accuracy, precision, recall and f1.
+    The overview of metric could be seen at the document of `paddle.metric
+    <https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/metric/Overview_cn.html>`_ 
```

Comment:
The overview of metric -> The overview of all metrics

Reply:
Thanks. Done.